src/components/StudioList.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+import CardComponent from "./CardComponent";
+
+const List = styled.section`
+  flex-grow: 1;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: transparent;
+`;
+
+// Mapping happens in this file//
+export default function StudioList() {
+  return (
+    <List>
+      <CardComponent />
```

Comment:
If you need time to implement the map-function that´s fine, but you should simulate the function by giving the components a variable based on which the Cards content is generated. This wa you will know how to construct the map-function.

Reply:
I dont get this point. I ask you in Person!